Project-1/README.md

```diff
@@ -14,62 +14,225 @@ Note: The standard interpretation of the logical symbols - "∨", "∧", "→",
+  Male: (x)
   (b)  A that says that x is an aunt of y
+  A: Parent_of: (zw, zx, wy) ~Male^(∃parent_of.(∃parent_of>=1).^~∃parent_of.(>=1).)
   (c)  C that says that x and y are cousins
-  (d)  O that says that x is an only child  
+  C: parent_of: (zx, wy, vw, vz) ∃Parent_of.(∃parent_of>=2).^∃Parent_of.(∃parent_of>=1).^∃Parent_of.(∃parent_of>=1).
+  (d)  O that says that x is an only child 
+  O= parent_of (∃parent_of=1) 
   (e)  T that says that x has exactly two brothers 
+  T= Parent_of: (wx,wy,wz) ∃parent_of.(∃Parent_of=3)^Male(∃Male>=2)
+  Male: (y,z)
   ```
 
 
 5. Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both. 
+B= Male^∃Parent_of.(∃parent_of>=2).
+O= parent_of (∃parent_of=1)
+O^~B
+B^~O
+Jason: B
+(Mary Ann, Jason)
+(Mary Ann, Mona)
+Killian: O
+Amanda: Parent_of Killian
+Jason: (w,B)
+Mary Ann: Z
+Mona: Y
+Killian: (O,x)
+Amanda: v
+parent: (z,v)
+brother: w
+Only child: x
+child: w,y,x
+parent_of: zw,zy,vx
+Male: x,w
 
 6. Explain the difference - using natural language - between the first-order prefixes:
   ```
   (a) ∃x∀y and ∀x∃y
+  The first claims that for something such that it is x, every thing such that it is Y has a certain relationship with it. The second claims that for all things such that they are X, there is some Y that has a certain relationship with it.
   (b) ∃x∀y∃z and ∀x∃y∀z 
+  The first claims that for something such that it is x, everything such that it is Y has a certain relationship with X and that there is something such that it is Z has a certain relationship with all Ys and some X. The second claims that for all things such that they are X, there is something such that it is Y that has a certain relationship with it and everything such that it is Z has a certain relationship with some Y and everything such that it is X.
   (c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
+  The first claims that for all things such that they are x, there is something such that is is Y in a certain relationship with it and everything that is such that it is z is in a certain relationship with all xs and some y and there is some w which is in a certain relationship with all zs and some y and all xs. In the second, it claims that there is something such that it is x where there all ys are in a certain relationship with it, and there is some z which has a certain relationship with some x and all ys and all things such that they are w are in a certain relationship with some z and all ys and some x.
 ```
 	
 7. Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
 ```
 ∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+X1>Y1, X2>Z1<Y2, X2>Y2 Arrows indicate relationship R and its directionality
 ∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+X1>Y1>Z1, X2>Z2
```

Comment:
How should these graphs be interpreted?

Reply:
The arrows indicate the direction of the R relationship such that X1 has the R relationship with Y1 which has the R relationship with Z1.